docs/3.0/deploy/infrastructure-examples/docker.mdx

```diff
@@ -50,6 +65,12 @@ It's now polling the Prefect API to check for any scheduled flow runs it should
 You'll see your new worker listed in the UI under the **Workers** tab of the Work Pools page with a recent last polled date.
 The work pool should have a `Ready` status indicator.
 
+<Tip>
+**Pro Tip:**
+
+If `my-docker-pool` does not already exist, the above command will create it for you automatically with the default settings for that work pool type, in this case `docker`.
```

Comment:
For this to be true, you need to provide `--type` to the `worker start` command.